src/core/include/rs/feature_structure_proxy.h

```diff
@@ -662,6 +697,42 @@ class ListFeatureStructureEntry : public FeatureStructureEntry<std::vector<T, TA
     return success;
   }
 
+  template<typename TargetT, typename AnnotT>
+  bool filter(std::vector<TargetT> &result, std::vector<std::vector<AnnotT>> &annots)
```

Comment:
In line 3 what is rackA and rackC? I guess you mean toolA and toolB. Could you add this as comments to the code pls. And change the example to something that is in RoboSherlock by default.  (e.g. ```rs::Classification```)